en/core-libraries/email.rst

```diff
@@ -156,7 +156,10 @@ The above configuration would enable TLS communication for email messages.
     account <https://support.google.com/accounts/answer/6010255>`__.
 
 .. note::
-
+    `Use Gmail SMTP settings to send mail from a printer, scanner, or app     
+    <https://support.google.com/a/answer/176600?hl=en>`.
```

Comment:
Why is printer/scanner relevant here?

Reply:
Link needs to end with `__` to be valid.

```rst
`Link text <URL>`__
```

Is the correct formatting for inline external links.